pages/linux/scrontab.md

```diff
@@ -0,0 +1,20 @@
+# scrontab
+
+> Manage Slurm crontab files.
+> More information: <https://slurm.schedmd.com/scrontab.html>.
+
+- Install a new crontab from the specified file:
+
+`scrontab {{path/to/file}}`
+
+- [e]dit the crontab of the specified user:
+
+`scrontab --user={{user_id}} -e`
```

Comment:
Should we also have an example here for interactively editing the crontab of the current user?

Reply:
You're right, i'll add one.